otter/test/convergence/test_effecting.py

```diff
@@ -112,3 +114,36 @@ def test_multiple_requests_of_different_type(self):
                              headers=None,
                              data=data_sentinel)]
         self.assertCompileTo(conv_requests, expected_effects)
+
+
+@implementer(IStep)
+class Steppy(object):
+    """A dummy step."""
+    def as_request(self):
+        """Return a simple GET .../whatever on CLOUD_LOAD_BALANCERS"""
+        return Request(service=ServiceType.CLOUD_LOAD_BALANCERS,
+                       method="GET",
+                       path="whatever")
+
+
+class StepsToEffectTests(SynchronousTestCase):
```

Comment:
I think it's reasonable. It's only testing one case, just to make sure that it's likely to be composed correctly.